source/usage-examples.txt

```diff
@@ -61,6 +61,36 @@ with your instance of MongoDB:
    const uri =
       "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority";
 
+All examples use ES module imports. You 
+`must enable ES module imports <https://nodejs.org/api/esm.html#esm_enabling>`__
+by adding the following key-value pair to your package.json file:
```

Comment:
Maybe just be more clear about what the user can do:


> All examples use ES module imports, however any example can be used with a commonJS require. Copy the import statement below that matches your environment. If you want to use ES modules in Node.js see the documentation here: [LINK to Node.js docs].


The `"type": "module"` is only one way, and the other would be to use the `.mjs` extension, since there are multiple ways and other implications that come with adopting ES modules, we should leave it to the Node.js docs entirely to cover.

Reply:
I agree that the current language is too restrictive, as there are multiple ways, however I think the docs should give enough guidance to get the examples running without readers having to leave the page.

I think if we present steps to make the example work we lift the burden of making readers have to become familiar with the node.js docs or consider whether to use `.mjs` extensions or update package.json (both work, so I think if we just pick one I think it makes it easier for them).

How is "You can enable ES modules..."?

I think in the docs we try to put alternative paths in notes so that readers have a clear idea of what they need to do to get to a base working state (even if both choices give the same result, it makes it harder for readers to get to a base working state if they have to make a choice). 

How is updating the note to communicate the information above? Update: 


>    You can use any usage example with CommonJS require. To use CommonJS require, you
>    must swap out the ES module import statement for your CommonJS require
>    statement.
>    
>    Click on the tabs to see the syntax for importing the driver with ES module
>    imports and CommonJS requires:

Reply:
I agree with this approach, thanks for taking the time to rephrase it!